.kokoro/build.sh

```diff
@@ -31,7 +31,7 @@ env | grep KOKORO
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
 
 # Setup project id.
-export PROJECT_ID=$(cat "${KOKORO_GFILE_DIR}/project-id.txt")
+export PROJECT_ID=$(cat "${KOKORO_GFILE_DIR}/project-id.json")
```

Comment:
`cat: /tmpfs/src/gfile/project-id.json: No such file or directory` I think it was fine before when you used `project-id.txt`?

Reply:
@dandhlee That filename works for the main Kokoro build, but *not* for the `docs-presubmit` build (see [this failure](https://source.cloud.google.com/results/invocations/a3937c80-4f2f-48a0-a9cf-d6dfa090d34a/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fgoogle-auth-library-python%2Fdocs%2Fdocs-presubmit/log)).

Maybe there is something weird in the `google3` Kokoro config for this repository?  I honestly can't figure out why `docs-presubmit` is failing here.